docs/REST API/rest-api.md

```diff
@@ -4860,17 +4841,18 @@ solely on the response status code returned by this endpoint.
 
 #### Example
 
-`DELETE /api/v1/fleet/mdm/profiles/f663713f-04ee-40f0-a95a-7af428c351a9`
+`DELETE /api/v1/fleet/configuration_profiles/f663713f-04ee-40f0-a95a-7af428c351a9`
 
 ##### Default response
 
 `Status: 200`
 
+
 ### Update disk encryption enforcement
 
 _Available in Fleet Premium_
 
-`PATCH /api/v1/fleet/mdm/apple/settings`
+`POST /api/v1/fleet/disk_encryption`
```

Comment:
This was also in this PR: https://github.com/fleetdm/fleet/pull/16129/files

TODO

Reply:
never mind! I think I knew about the change & handled this ahead of time, vaguely remember that now